app/models/miq_worker/container_common.rb

```diff
@@ -44,15 +44,28 @@ def default_image
     def resource_constraints
       mem_threshold = self.class.worker_settings[:memory_threshold]
       cpu_threshold = self.class.worker_settings[:cpu_threshold_percent]
+      mem_default   = self.class.worker_settings[:memory_minimum]
+      cpu_default   = self.class.worker_settings[:cpu_minimum_percent]
 
       return {} if !Settings.server.worker_monitor.enforce_resource_constraints || (mem_threshold.nil? && cpu_threshold.nil?)
 
       {:limits => {}}.tap do |h|
-        h[:limits][:memory] = "#{mem_threshold / 1.megabyte}Mi" if mem_threshold
+        if mem_threshold
+          h[:limits][:memory] = format_memory_threshold(mem_threshold)
+          mem_default ||= 256
+        end
+
         if cpu_threshold
-          millicores = ((cpu_threshold / 100.0) * 1000).to_i
-          h[:limits][:cpu] = "#{millicores}m"
+          h[:limits][:cpu] = format_cpu_threshold(cpu_threshold)
+          cpu_default ||= 0.5
         end
+
+        if mem_default
+          h[:defaultRequest][:memory] = format_memory_threshold(mem_default)
+          mem_default ||= 256
```

Comment:
hehe, should these move into settings.yml, see above...

Reply:
I'm not sure I understand this... how can this line ever execute since we're inside an `if mem_default` block